src/build.rs

```diff
@@ -157,11 +157,22 @@ pub fn build_get_nso(args: Vec<String>) -> Result<PathBuf> {
     Ok(nso_path)
 }
 
-pub fn build(mut args: Vec<String>, release: bool, nso: bool) -> Result<()> {
+pub fn build(mut args: Vec<String>, release: bool, nso: bool, features: Vec<String>) -> Result<()> {
     if release {
         args.push("--release".into());
     }
 
+    if !features.is_empty() {
+        args.push("--features".into());
+        let feature_count = features.len();
+        for (num, feature) in features.into_iter().enumerate() {
+            args.push(feature);
+            if num != (feature_count - 1) {
+                args.push(",".into());
+            }
+        }
```

Comment:
Probably use [join](https://doc.rust-lang.org/std/primitive.slice.html#method.join)?

i.e. since you have a list of features you could do something like:

```rust
args.push(format!("--features={}", features.join(",")));
```